src/main/java/io/reactorsolutions/vertx_kafka/verticles/BatchConsumerVerticle.java

```diff
@@ -0,0 +1,40 @@
+package io.reactorsolutions.vertx_kafka.verticles;
+
+import io.reactorsolutions.vertx_kafka.config.consumer.ConsumerOptions;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Context;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.kafka.client.consumer.KafkaConsumer;
+import io.vertx.kafka.client.consumer.KafkaConsumerRecord;
+import io.vertx.kafka.client.consumer.KafkaConsumerRecords;
+
+public class BatchConsumerVerticle extends AbstractVerticle {
+  KafkaConsumer<String, JsonObject> consumer;
+
+
+  @Override
+  public void init(Vertx vertx, Context context) {
+    super.init(vertx, context);
+    consumer = KafkaConsumer.create(vertx,new ConsumerOptions().getConfig());
+  }
+
+  @Override
+  public void start(Promise<Void> startPromise) throws Exception {
+    consumer.handler(record->{});
```

Comment:
I would expand this example adding "logic" in the single record handler, so we see that both handlers are called.